### PR TITLE
Updating dotnet functional tests to print error to console if any

### PR DESCRIPTION
**Pull request by issacnitin**

### CODE REVIEW

File: dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs [hunk 1]
```diff
@@ -47,12 +46,14 @@ public void SampleFunctionalTest1()
...
                     break;
                 }
-                catch
+                catch(Exception e)
                 {
                     var currentTimestamp = DateTime.Now.Millisecond;
                     if(currentTimestamp > endTimestamp)
                     {
-                        throw;
+                        Console.Write("##vso[task.logissue type=error;]Test SampleFunctionalTest1 failed with error: " + e.ToString());
```
Comment (sachinma) [inline dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs]: let not have task specific constructs in the code

Comment (issacnitin) [reply dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs]: Without this we won't get logs in Kusto.. Whatever we write to stderr goes to test results directly and skips logging to console, so this is how we can print to both console and test results

File: dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs [hunk 2]
```diff
@@ -47,12 +46,14 @@ public void SampleFunctionalTest1()
...
+                        throw e;
```
Comment (bishal-pdMSFT) [inline dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs]: you can just do "throw" without "e"

File: dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs [hunk 3]
```diff
@@ -47,12 +46,14 @@ public void SampleFunctionalTest1()
...
+                        break;
```
Comment (bishal-pdMSFT) [inline dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs]: why break is needed?

Comment (issacnitin) [reply dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs]: It doesn't seem to break out of the loop without break statement.. the test will continue running indefinitely without break

Comment (issacnitin) [reply dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs]: I've updated

**Pull request merged**